docs/guide/http_cache.md

```diff
@@ -219,6 +219,11 @@ ezpublish:
                 purge_servers: ["http://varnish.server1", "http://varnish.server2", "http://varnish.server3"]
 ```
 
+!!! tip "Environment Variables"
+
+    eZ Platform is also using Environment Variables by default, then you can define those values in the Environment.
```

Comment:
Maybe formulate it a bit more simply:
`eZ Platform uses environment variables by default, so you can define those values in the environment.`